optimas/diagnostics/exploration_diagnostics.py

```diff
@@ -100,7 +106,30 @@ def _create_exploration(
             ),
             evaluator=Evaluator(sim_function=None),
             history=history_path,
+            exploration_dir_path=exploration_dir_path,
+        )
+
+    def _create_sim_dir_paths(self):
+        """Create a dict with the path to the sim dir of each evaluation."""
+        self._sim_dir_paths = {}
+        ensemble_dir_path = os.path.join(
+            self._exploration.exploration_dir_path, "evaluations"
         )
+        if os.path.isdir(ensemble_dir_path):
+            sim_dirs = os.listdir(ensemble_dir_path)
+            for sim_dir in sim_dirs:
+                if sim_dir.startswith("sim"):
+                    try:
+                        trial_index = int(sim_dir[3:])
```

Comment:
Very minor: 
This removes the `"sim"` part of the simulation directory name and converts the rest to an integer which matches the trial_index by construction.
Maybe is clearer is we do: `trial_index = int(sim_dir.lstrip("sim"))`